src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -0,0 +1,99 @@
+# Secure Communications
+
+Here we describe how to setup the Secure Communications (`Secure Comms` for short) feature of the Adaptor and Forwarder
+
+Secure Comms feature establish an ssh channel by the Adaptor as an ssh client to the Forwarder as an ssh server. The ssh channel is than used for all control communication between the Adaptor and Forwarder. Secure Comms use a two phase approach:
+- Attestation Phase: During this phase an ssh channel is created to enable the Peer Pod to attest against an attestation service. The attestation traffic is tunneled from the Peer Pod to the CAA and from there sent to a KBS as configured at the CAA. Once attestation is achieved, the Forwarder obtains Peer Pod keys from the KBS. The Peer Pod keys are dynamically added to the KBS by the CAA prior to starting the Peer Pod. 
+- Kubernetes Phase: During this phase an ssh channel is created to enable Runtime communicate with teh Kata Agent. This phase uses the keys obtained during the Attestation Phase ensuring that the communication is secured. 
+
+Additional tunnels can be establish via the ssh channel by configuring the Adaptor and Forwarder as described below.
+
+## Setup
+
+### Deploy CAA
+Use any of the option for installing CAA depending on the cloud driver used.
+
+### Deploy KBS-Operator
+Deploy KBS-Operator by following instructions at [KBS Operator Getting Started](https://github.com/confidential-containers/kbs-operator?tab=readme-ov-file#getting-started).
+
+Copy the kbs-client secret from the `kbs-operator-system` namespace to the `confidential-containers-system` ns. This can be done using:
+
+```sh
+kubectl get secret kbs-client -n kbs-operator-system -o json|jq --arg ns "confidential-containers-system" 'del(.metadata["creationTimestamp","resourceVersion","selfLink","uid","annotations"]) | .metadata.namespace |= $ns' |kubectl apply -f -
+```
+
+For a testing environment, you may need to change the policy of the KBS and AS using the KBS Client to allow all or fit your own policy. One way to do that is:
+
+```sh
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container as -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opt/confidential-containers/attestation-service/opa/default.rego // replace default policy to true
+
+kubectl -n kbs-operator-system exec deployment/kbs-deployment --container kbs -it -- /bin/bash
+        apt update
+        apt install vim
+        vim /opa/confidential-containers/kbs/policy.rego // replace default policy to true
+```
+
+### Build a podvm that enforces Secure-Comms
```

Comment:
Should this part go at the top of the Setup - I might be in the minority, but I'd find it easier to build the podvm before I create a cluster and then have to edit it?

Reply:
I am fine with any order people feel comfortable with :) 

The natural way for me is to setup a cluster (and its services) before I get to build the image of the podvm that will be later deployed using that cluster. This gives me a sense of grounding... ;)

Reply:
Fair enough. Once we can avoid the podvm needing rebuilding later that will simplify the flow a lot anyway, so I'm not against leaving it as is.